### DISCUSSION TITLE
成都招聘IPFS，欢迎志同道合的小伙伴加入~~技术总监，开发工程师（Go，Rust），运维工程师

### DISCUSSION THREAD

**Discussion by siyadong1** [filecoin-project/community-china#133] [人才招聘]
### 技术总监：
1.我们希望你有带团队的能力
2.我们希望你有极强的责任心
3.我们希望你有5P+的IPFS集群经验
4.待遇：面谈。只要能力强，待遇不是问题。

### Go工程师：
1.需要熟练使用Go语言，有区块链领域项目实战经验，我们希望你有自我驱动力和较强的自学能力。
2.加分项：有IPFS开发经验。
3.待遇：15K-25K，有能力则上不封顶。

### Rust工程师：
1.熟练使用Rust，掌握算法优化，对区块链了解，我们希望你有自我驱动力和较强的学习能力。
2.加分项：有IPFS开发经验。
3.待遇：15K-25K，有能力则上不封顶。

### 运维工程师：
1.对服务器组装，硬件架构熟悉
2.对IDC机房实施安装有实战经验
3.掌握分布式存储的搭建和维护
4.了解区块链方面的知识
5.抗压能力强，学习能力强，善于与团队成员沟通。
6.加分项：有IPFS运维经验。
7.待遇：15K-25K，有能力则上不封顶。

**Comment by luoweidd** [comment]
毛遂自荐一下，最高，7p经验，多大10个节点以上，微信L_vgaigai

**Reply by luoweidd** [reply]
应征一下你家的技术总监~

**Reply by KaixuanZhao** [reply]
IPFS主网最大雅典娜矿池，提供一站式采购托管运营，fil存量数据迁移出海等服务，全网算力领先1400PB，需要B端1PB以上，C端50T，100T，200T（可扩容）和FIL存量数据迁移出海服务方案的滴滴我，欢迎加入社群Wechet：kkmmjjnn00

**Comment by GZpowerleader** [comment]
filecoin矿机软硬一体方案 学习交流滴滴我V18589289061

**Comment by luoweidd** [comment]
你们现在还在作么？



巴斯沃形象科技技术工作室    Baswo ITT Studio
 ----------------------------------------------------------------------------------------
 主营：企业网络外包服务、网络集成安装，安防监控安装,光纤放装、基站、室分、宏站建设热冷接光纤，软件开发，网络建站，图像、logo、VI、UI设计定制，网络维护，电脑软硬件维护、销售、DIY、数据恢复，数码产品维护、刷机、销售，视频、广告、音频拍摄录制编辑、剪辑，服务器组装、服务器软硬件环境搭配、DIY；
 ----------------------------------------------------------------------------------------
 联系方式（phone）：18681781845
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; （Tel）:023-63841084
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; QQ:3245554
 邮箱（Email）: ***@***.***
 ***@***.***
 网址（Web）&nbsp;：www.baswo.com（未申请到Ip，维护中……）




&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "filecoin-project/community-china"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年10月13日(星期三) 中午11:17
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [filecoin-project/community-china] 成都招聘IPFS，欢迎志同道合的小伙伴加入~~技术总监，开发工程师（Go，Rust），运维工程师 (Discussion #133)





 
filecoin矿机软硬一体方案 学习交流滴滴我V18589289061
 
—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by yunfens** [comment]
由于急用钱资金链断裂，项目整体转让，机房自建配置，3台空调，50千瓦发电机，30千瓦ups；存储配置：6台36盘14T存储机外加5台同配置空的存储机（不带硬盘），5台封装机，每天10T的封装能力，并且已经封装了430T，一台节点机，7张3090显卡，算力均为1t内存，16tp4510缓存盘，3090显卡adm7402，可以公司一起转，另可以免费提供技术支持！机房租金3000每月，可放置6台42U机柜，并可以扩充更多！

**Comment by huchenghuang7** [comment]
您好，请问现在还招聘运维工程师么？坐标成都

**Comment by luhong123** [comment]
我来人在深圳

**Comment by luoweidd** [comment]
？？？



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年2月6日(周一) 晚上6:07
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [filecoin-project/community-china] 成都招聘IPFS，欢迎志同道合的小伙伴加入~~技术总监，开发工程师（Go，Rust），运维工程师 (Discussion #133)




 
我来人在深圳

**Comment by huchenghuang7** [comment]
你好，请问成都IPFS有招人吗？我想应聘。谢谢！

发自 Outlook<http://aka.ms/weboutlook>
________________________________
发件人: luoweidd ***@***.***>
发送时间: 2023年3月7日 14:24
收件人: filecoin-project/community-china ***@***.***>
抄送: huchenghuang7 ***@***.***>; Comment ***@***.***>
主题: Re: [filecoin-project/community-china] 成都招聘IPFS，欢迎志同道合的小伙伴加入~~技术总监，开发工程师（Go，Rust），运维工程师 (Discussion #133)


？？？



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年2月6日(周一) 晚上6:07
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [filecoin-project/community-china] 成都招聘IPFS，欢迎志同道合的小伙伴加入~~技术总监，开发工程师（Go，Rust），运维工程师 (Discussion #133)





我来人在深圳

―
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you commented.Message ID: ***@***.***&gt;

―

**Comment by luoweidd** [comment]
哈哈~ 都是找活干的~，有没有老板搭节点的？全程技术支持



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年3月8日(周三) 晚上10:33
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [filecoin-project/community-china] 成都招聘IPFS，欢迎志同道合的小伙伴加入~~技术总监，开发工程师（Go，Rust），运维工程师 (Discussion #133)




 你好，请问成都IPFS有招人吗？我想应聘。谢谢！ 
 
 发自 Outlook<http://aka.ms/weboutlook&gt; 
 ________________________________ 
 发件人: luoweidd ***@***.***&gt; 
 发送时间: 2023年3月7日 14:24 
 收件人: filecoin-project/community-china ***@***.***&gt; 
 抄送: huchenghuang7 ***@***.***&gt;; Comment ***@***.***&gt; 
 主题: Re: [filecoin-project/community-china] 成都招聘IPFS，欢迎志同道合的小伙伴加入~~技术总监，开发工程师（Go，Rust），运维工程师 (Discussion #133) 
 
 
 ？？？ 
 
 
 
 ---原始邮件--- 
 发件人: ***@***.***&amp;gt; 
 发送时间: 2023年2月6日(周一) 晚上6:07 
 收件人: ***@***.***&amp;gt;; 
 抄送: ***@***.******@***.***&amp;gt;; 
 主题: Re: [filecoin-project/community-china] 成都招聘IPFS，欢迎志同道合的小伙伴加入~~技术总监，开发工程师（Go，Rust），运维工程师 (Discussion #133) 
 
 
 
 
 
 我来人在深圳 
 
 ― 
 Reply to this email directly, view it on GitHub, or unsubscribe. 
 You are receiving this because you commented.Message ID: ***@***.***&amp;gt; 
 
 ―

**Comment by Ipfs968fil** [comment]
什么都不会，但我热爱学习，可以吗？

**Comment by Ipfs968fil** [comment]
V13033441155
